packages/desktop-client/src/components/modals/GoCardlessExternalMsg.tsx

```diff
@@ -140,7 +155,6 @@ export default function GoCardlessExternalMsg({
           <Autocomplete
             strict
             highlightFirst
-            disabled={isConfigurationLoading}
```

Comment:
Not a valid prop on autocomplete. If we want to disable it we set the "inputProps" prop